kodilla-patterns2/src/main/java/com/kodilla/kodillapatterns2/observer/homework/ExerciseOne.java

```diff
@@ -0,0 +1,9 @@
+package com.kodilla.kodillapatterns2.observer.homework;
+
+public class ExerciseOne extends ExercisesQueue {
```

Comment:
Nie rozumiem sensu tych klas: `ExcersiseOne` oraz `ExcersiseTwo`. W jakim celu je wprowadzasz?
Jeśli już, to tak jak pisałem w innym komentarzu - każdy obiekt typu `ExcersisesQueue` reprezentuje zadania jednego kursana. W teście jest już OK, tylko nie wiem po co Ci te dodatkowe klasy...